internal/ad/backends/winbind/mock/libwbclient_mock.c

```diff
@@ -0,0 +1,61 @@
+#include <stdlib.h>
+#include <string.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdio.h>
+
+#include <wbclient.h>
+
+char *get_mock_behavior()
+{
+    char *behavior = getenv("ADSYS_WBCLIENT_BEHAVIOR");
+    if (behavior == NULL) {
+        printf("ADSYS_WBCLIENT_BEHAVIOR not set, exiting...");
+        exit(1);
```

Comment:
I feel that if we forget to set it in some tests, it will just exit, correct? Is there a risk that if we come back to it in few months time, we will scratch our head on why the test program just exits?

Reply:
this is fine for me